release/staging-resources-cdk/README.md

```diff
@@ -53,6 +53,12 @@ The following CDK commands all require defining context. The context variables a
 * `archivesBucketName` - The name of the S3 bucket you will use to deploy. This bucket must be in the same region as your stack.
 * `dataPrepperOrganization` - The name of the GitHub organization which has the `data-prepper` repository. This allows you to create staging environments for forks. The default value is `opensearch-project`.
 
+(Optional) If you need a new S3 bucket:
+
+```
+cdk deploy ArtifactsBucketStack --context archivesBucketName={s3-bucket-name} --context dataPrepperOrganization={data-prepper-organization-name}
```

Comment:
`dataPrepperOrganization`  is never used in this stack.

Reply:
I removed this from the documentation, so it should be obsolete.